src/app/notifications/components/SearchFailure/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { memo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { assertWindow, useOnDOMEvent, useTimeout } from '../../../utils';
+import { Header } from '../Card';
+import {
+  BannerBody,
+  BannerBodyWrapper,
+  clearErrorAfter,
+  CloseButton,
+  CloseIcon
+} from './styles';
+
+interface Props {
+  dismiss: () => void;
+}
+
+const window = assertWindow();
```

Comment:
this is gonna break everything if you put it up here, put it down in the function body and it will work i think